docs/UserGuide.md

```diff
@@ -234,28 +263,112 @@ Oops! There seems to be an error, please check your fields again.
+Charlotte Oliveiro; Phone: 93210283; Email: charlotte@example.com; Address: Blk 11 Ang Mo Kio Street 74, #11-04; Housing: HDB; Availability: Available; Animal name: nil; Animal type: able.Dog; Tags: [new], 
+Roy Balakrishnan; Phone: 92624417; Email: royb@example.com; Address: Blk 45 Aljunied Street 85, #11-31; Housing: Landed; Availability: Available; Animal name: nil; Animal type: able.Cat; Tags: [new]
 ```
 
 Expected output (fail):
+
+1. A non-positive index provided: `delete -1`
 ```agsl
-Oops! Invalid fosterer index provided, please check again.
+Invalid command format! 
+delete: Deletes fosterers identified by the index number used in the displayed person list.
+Parameters: INDEX [INDEX...] (must be a positive integer)
+Example: delete 1 2 3
 ```
+
+2. An index exceeds the number of fosterers in the list: `delete 0 1 99` but the list only has 6 fosterers
+```agsl
+The fosterer index provided is invalid
+```
+
+### Viewing statistics : `stats`
+Displays the requested statistic about the currently displayed list.
+
+Format: `stats FIELD`
+
+Parameters:`FIELD` 
+* The requested field whose statistic is to be calculated.
+* Valid fields that can be requested for are:
+  * `avail` :  shows the statistics of fosterers  who are available to foster, and the animals they can foster.
+  * `current` : shows the statistics of fosterers who are currently fostering, and the animals they are fostering.
+  * `housing` : shows the statistics of the various housing types.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+Statistics are calculated based on the currently displayed list. If you performed a `find available` command before `stats avail`, the resulting statistic will show that 100% of listed fosterers are available. Hence, please ensure that the current list for fosterers is the desired list you want your statistic to be calculated from. 
+</div>
+
+Examples: 
+* `list` followed by `stats avail` calculates the statistics of fosterers who are available, based on all fosterers in the address book.
+* `find cat` followed by `stats avail` calculates the statistics of fosterers who are available, based on the fosterers who are either currently fostering a cat or are able to foster a cat. 
+* `list` followed by `stats current` calculates the statistics of fosterers who are currently fostering, based on all fosterers in the address book.
+* `find available dog` followed by `stats housing` calculates the statistics of the various housing types based on the fosterers who are available and are able to foster a dog.
+
+Expected output (success):
+1. `list` followed by `stats avail`
+```agsl
+6 out of 10 listed fosterers are available (60%)!
+Out of those available, 
+- 2 can foster dogs (33.3%)
+- 4 can foster cats (66.7%)
+```
+
+2. `list` followed by `stats current`
+```agsl
+4 out of 10 listed fosterers are currently fostering (40%)!
+Out of those fostering, 
+- 1 foster dogs (25.0%)
+- 3 foster cats (75.0%)
```

Comment:
Can have unknown here also i think (aka 'nil')

Reply:
actually,. you are right. i realised this flaw because what i did was check for those not available to count as those who are currently fostering. BUT THEN i realised there is the case where the user is not available, but they are also not fostering!!!!! thanks for pointing this out, i will fix my stats command. 
but for the output here, i will not be adding nil because the focus is on those who are currently fostering, so i will not be sharing the nil case with them. what do u think!

Reply:
yup, based on the new change in conditions for add this sounds right